src/main/resources/assets/rgen/blockstates/flower.json

```diff
@@ -1,5 +1,5 @@
 {
   "variants": {
-    "type=forget_me_not": { "model": "rgen:flower/forget_me_not" }
+    "type=forget_me_not": { "model": "rgen:flower/forgetmenot/forgetmenot" }
```

Comment:
pick `forget_me_not` or `forgetmenot`. i don't care which, but it should be consistent.

Reply:
for its in code name it will be `forgetmenot` however its en_us name will be `Forget-me-not` and its en_pt `Ne’er-Ferget Blossom`